django_business_metrics/_internal/metrics_manager.py

```diff
@@ -1,6 +1,9 @@
+from __future__ import annotations
```

Comment:
I might be mistaken about how this works. But does using this not mean that the code (e.g. `str | None`) will break for some Python 3.9 versions, e.g. 3.9.0? Or was `|` as `Union` introduced in `__future__.annotations` before that? I mean, it's probably fine either way but it might mean that we don't support Python 3.9 period, just the latter versions, could be mentioned in the changelog and `pyproject.toml`.

Reply:
nah, https://peps.python.org/pep-0585/ was introduced in python 3.9

I will remove this here since there is a chance that https://peps.python.org/pep-0563/ will be superseded https://peps.python.org/pep-0563/#rationale-and-goals

Reply:
oh wait, right, https://peps.python.org/pep-0604/ is from 3.10, and postponed type annotations allow us to use it anyway. Lets keep it.

As to support of 3.9 - we are explicitly required by our handbook to support it.

Reply:
Learning every day, I had a completely wrong idea how `from future import __annotations__` works. Sorry for wasting your time.

Reply:
it wasn't a waste I also learned something! thanks!